src/installer/bin.ts

```diff
@@ -1,5 +1,5 @@
 import chalk from 'chalk'
-import isCI from 'is-ci'
+import ci from 'ci-info'
```

Comment:
More of a preference really, but just pull this out during import?

```suggestion
import { isCI } from 'ci-info'
```

Reply:
I like it, updated!